src/interactive.sh

```diff
@@ -38,3 +38,94 @@ function check_ssh_installation()
   [[ -n "$cmd" ]]
 
 }
+
+function interactive_init()
+{
+
+  local cmd=''
+  local distro=''
+  local has_ssh=0
+  local ssh_keys
+  local key_name ssh_dir
+  local ssh_user ssh_ip
+
+  load_string "$KW_LIB_DIR/strings/init.txt"
+
+  say "${string_file['text_interactive_start']}"
+
+  printf "\n"
+  read -r -p '>>> Press ENTER to continue'
+
+  say "${string_file['text_ssh_start']}"
+  printf "\n"
+  read -r -p '>>> Press ENTER to continue'
+
+  check_ssh_installation
```

Comment:
At this point, you can expect that the ssh package is available.

Reply:
Why is that? Before this point, we have not yet done anything related to ssh. Is this being dealt with in another PR?